### PR TITLE
Add emailAddress, twitterHandle, description, descriptionXML, _imageURL

### PR DESCRIPTION
**Pull request by sjwells**
Changes required to support columnists

### CODE REVIEW

File: people/model.go [hunk 1]
```diff
@@ -5,12 +5,17 @@ import "sort"
...
-	Salutation  string       `json:"salutation,omitempty"`
-	Aliases     []string     `json:"aliases,omitempty"`
+	UUID           string       `json:"uuid"`
+	BirthYear      int          `json:"birthYear,omitempty"`
+	Identifiers    []identifier `json:"identifiers,omitempty"`
+	Name           string       `json:"name,omitempty"`
+	Salutation     string       `json:"salutation,omitempty"`
+	Aliases        []string     `json:"aliases,omitempty"`
+	EmailAddress   string       `json:"emailAddress,omitempty"`
+	TwitterHandle  string       `json:"twitterHandler,omitempty"`
```
Comment (kghatton) [inline people/model.go]: twitterHandler?

Comment (sjwells) [inline people/model.go]: Dammit - I have real typing issues with 'handle'. All those years of enterprise Java...

**Comment by scott-ace-newton** [comment]
:+1:

**Comment by kghatton** [comment]
:+1:

**Comment by izzyblues** [comment]
👍

**Pull request merged**